package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "pretest": "if [ ! -f ./config.js ]; then cp config_sample.js config.js; fi",
-    "test": "mocha --no-timeouts"
+    "test": "mocha --no-timeouts",
```

Comment:
@carloscdias What do you think about running `npm run lint` as part of the unit test too? For example:

`"test": "npm run lint && mocha --no-timeouts",`

Reply:
I think it's a good idea @jflory7 that way travisCI would let us know when a feature branch don't follow our lint's rules! Good one!